docs/operational_guide/monitoring.md

```diff
@@ -0,0 +1,74 @@
+## Metrics
+
+TODO
+
+## Logs
+
+TODO
+
+## Tracing
+
+M3DB is integrated with [opentracing](https://opentracing.io/) to provide
+insight into query performance and errors.
+
+### Configuration
+Currently, only https://www.jaegertracing.io/ is supported as a backend.
+
+To enable it, set tracing.backend to "jaeger":
+
+```
+tracing:
+    backend: jaeger  # enables jaeger with default configs
+    jaeger:
+        # optional configuration for jaeger -- see
+        # https://github.com/jaegertracing/jaeger-client-go/blob/master/config/config.go#L37
+        # for options
```

Comment:
Worth calling out that sampling is heavily encouraged? Otherwise it's likely to create a ton of data